contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -169,25 +169,25 @@ func (h *htlcIncomingContestResolver) Resolve() (ContractResolver, error) {
 
 	// Define closure to process htlc resolutions either direct or triggered by
 	// later notification.
-	processHodlEvent := func(e invoices.HtlcResolution) (ContractResolver,
-		error) {
+	processHtlcResolution :=
```

Comment:
style nit:
```go
processHtlcResolution := func(e invoices.HtlcResolution) (
       ContractResolver, error) {
```

Would that help with the extra identation also? 🤔

Reply:
yeah that's much better!